app/(main)/league/[leagueId]/entry/all/page.test.tsx

```diff
@@ -318,12 +340,159 @@ describe('League entries page (Entry Component)', () => {
+        week: 1,
+        selectedTeams: ['Packers', 'Bears'],
+      },
+    ]);
+    mockGetGameWeek.mockResolvedValueOnce({ week: 1 });
+    mockGetCurrentLeague.mockResolvedValue({
+      leagueName: 'Test League',
+      participants: 10,
+      survivors: 8,
+    });
+    mockGetNFLTeams.mockResolvedValue([
+      { teamId: 'packers', teamLogo: '/packers-logo.png', teamName: 'Packers' },
+      { teamId: 'bears', teamLogo: '/bears-logo.png', teamName: 'Bears' },
+    ]);
+
+    render(<Entry params={{ leagueId: '123' }} />);
+
+    await waitFor(() => {
+      expect(mockGetGameWeek).toHaveBeenCalled();
+      expect(mockGetCurrentUserEntries).toHaveBeenCalled();
+      expect(mockGetCurrentLeague).toHaveBeenCalled();
+      expect(mockGetNFLTeams).toHaveBeenCalled();
+    });
+
+    const userPickHistory = screen.queryByTestId('user-pick-history');
+    expect(userPickHistory).not.toBeInTheDocument();
+  });
+
+  it('should display the user pick history if the current week is greater than 1', async () => {
+    mockUseDataStore.mockReturnValue({
+      ...mockUseDataStore(),
+      currentWeek: 2,
+      NFLTeams: [
+        {
+          teamId: 'packers',
+          teamLogo: '/packers-logo.png',
+          teamName: 'Packers',
+        },
+        {
+          teamId: 'bears',
+          teamLogo: '/bears-logo.png',
+          teamName: 'Bears',
+        },
+      ],
+    });
+    mockGetCurrentUserEntries.mockResolvedValueOnce([
+      {
+        $id: '123',
+        name: 'Test Entry',
+        week: 2,
+        selectedTeams: ['Packers', 'Bears'],
+      },
+    ]);
+    mockGetGameWeek.mockResolvedValueOnce({ week: 2 });
+    mockGetCurrentLeague.mockResolvedValue({
+      leagueName: 'Test League',
+      participants: ['123', '456'],
+      survivors: ['123', '456'],
+    });
+    mockGetNFLTeams.mockResolvedValue([
+      { teamId: 'packers', teamLogo: '/packers-logo.png', teamName: 'Packers' },
+      { teamId: 'bears', teamLogo: '/bears-logo.png', teamName: 'Bears' },
+    ]);
+
+    render(<Entry params={{ leagueId: '123' }} />);
+
+    await waitFor(() => {
+      expect(mockGetGameWeek).toHaveBeenCalled();
+      expect(mockGetCurrentUserEntries).toHaveBeenCalled();
+      expect(mockGetCurrentLeague).toHaveBeenCalled();
+      expect(mockGetNFLTeams).toHaveBeenCalled();
+    });
+    // Add a delay to allow for any asynchronous rendering
+    await new Promise((resolve) => setTimeout(resolve, 0));
+
+    const userPickHistory = screen.queryByTestId('user-pick-history');
+    expect(userPickHistory).toBeInTheDocument();
+    expect(screen.getByTestId('league-history-logo')).toHaveAttribute(
+      'src',
+      '/_next/image?url=%2Fpackers-logo.png&w=96&q=75',
+    );
+    expect(screen.getByTestId('league-entry-logo')).toHaveAttribute(
+      'src',
+      '/_next/image?url=%2Fbears-logo.png&w=96&q=75',
+    );
+  });
+
+  it('should not display the user pick history if the entries selected teams are empty', async () => {
```

Comment:
If we're using `test`, this should be changed to `test`, as well as all the other `it()` in this PR.

Reply:
@shashilo looking for direction here.